### PR TITLE
fix(ci/license-scanning): for trivy scanning you need trivy

### PR DESCRIPTION
**Pull request by marvinWolff**

### CODE REVIEW

File: .github/workflows/check-licenses.yaml [hunk 1]
```diff
@@ -22,4 +22,7 @@ jobs:
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
       - run: pip install yq
+      - run: |
+          wget https://github.com/aquasecurity/trivy/releases/download/v0.18.3/trivy_0.18.3_Linux-64bit.deb
+          sudo dpkg -i trivy_0.18.3_Linux-64bit.deb
```
Comment (cwrau) [inline .github/workflows/check-licenses.yaml:27]:
```suggestion
      - run: /home/linuxbrew/.linuxbrew/bin/brew install trivy
```

[the remark this answers is not in the document]
Comment (marvinWolff) [reply .github/workflows/check-licenses.yaml:27]: Does that just work?

[the remark this answers is not in the document]
Comment (cwrau) [reply .github/workflows/check-licenses.yaml:27]: Yes, see https://github.com/teutonet/teutonet-helm-charts/blob/main/.github/workflows/scan-for-cves.yaml?plain=1#L26

[the remark this answers is not in the document]
Comment (marvinWolff) [reply .github/workflows/check-licenses.yaml:27]: Crazy. Pretty cool tho

**Review by tasches — APPROVED** [review]

**Review by cwrau — APPROVED** [review]

**Pull request merged**